pkgs/by-name/uh/uhd/package.nix

```diff
@@ -176,6 +176,10 @@ stdenv.mkDerivation (finalAttrs: {
       dpdk
     ];
 
+  patches = [
+    ./fix-pkg-config.patch
```

Comment:
@jakobkukla well done for the work and the upstream submitted patch. Next time, Add a comment in the Nix code near the patch to the [PR for upstream](https://github.com/EttusResearch/uhd/pull/872).

Reply:
Will do!